js/flatbuffers.js

```diff
@@ -226,6 +226,18 @@ flatbuffers.Builder = function(opt_initial_size) {
   this.force_defaults = false;
```

Comment:
might be cleaner to reset this one also? if you clear a data structure, you expect it to be functionally equivalent to creating a new one. So it shouldn't retain state.

Reply:
Good catch, thanks.